multi-ai-cli/backends/lightwrap.py

```
@@ -0,0 +1,183 @@
+            "submit": {"candidates": self._selector_values(submit_selectors, "submit")},
+            "response": {"candidates": self._selector_values(response_selectors, "response")},
+            "ready": {"candidates": self._selector_values(ready_selectors, "ready")},
+            "login": {"candidates": list(login_selectors)},
+        }
+        path = self._local_config_path()
+        temporary = path.with_suffix(".tmp")
+        temporary.write_text(json.dumps(payload, indent=2) + "\n", encoding="utf-8")
+        try:
+            temporary.chmod(0o600)
+        except OSError:
+            pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)